sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/Config/SignalROptions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Azure.SignalR;
+using Microsoft.Azure.SignalR.Management;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    /// <summary>
+    /// Configuration options for SignalR extensions.
+    /// </summary>
+    /// <remarks>
+    /// Most properties here can be configured in <see cref="IConfiguration"/>, except JSON serialization.
+    /// </remarks>
+    public class SignalROptions
```

Comment:
Is this meant to be configured through a host.json file? 
/cc @mattchenderson

Reply:
No, I think users want to customize JSON serialization via C# code, for example, setting the `JsonSerializerSettings`, which is not convenient to configured through JSON or other files.

Reply:
Well, that is just one piece of the options - the other pieces can be configured in the host.json. It is similar for the Service Bus extension where some things need to be configured through code, but everything else can be configured in host.json.